lib/common/_data.go

```diff
@@ -0,0 +1,15 @@
+package sebakcommon
```

Comment:
I wish the code in this file, `_data.go` can be moved to `data.go`. Is there any reason to make new file?

Reply:
There was a file named `data.go` previously. I think the simplest solution would be to keep the previous file as is, and use `amount.go` instead of `data.go` for the new file. @kfangw ?